root/edit/cannot_vote.tt

```diff
@@ -0,0 +1,8 @@
+[% WRAPPER 'layout.tt' full_width=1 title=l('Error Voting Edit') %]
+    <h1>[% l('Error Voting Edit') %]</h1>
+    <p>
+        [% l('You’re currently not allowed to vote because an admin has revoked your privileges.
```

Comment:
Oops, the message is actually displayed twice. I simplified the message instead in https://github.com/metabrainz/musicbrainz-server/pull/524/commits/03f2172b2abcaed0661ce723dede01c363d59217